purchase_request/migrations/12.0.1.0.0/pre-migration.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2018-2019 Eficent Business and IT Consulting Services S.L.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0).
+
+import logging
+logger = logging.getLogger(__name__)
+
+
+def store_field_qty_in_progress(cr):
+    cr.execute("""SELECT column_name
+    FROM information_schema.columns
+    WHERE table_name='purchase_request_line' AND
+    column_name='qty_in_progress'""")
+    if not cr.fetchone():
+        cr.execute(
+            """
+            ALTER TABLE purchase_request_line ADD COLUMN qty_in_progress float;
+            """)
+
+    logger.info('Computing field qty_in_progress on purchase.request.line')
+
+    cr.execute(
+        """
+        UPDATE purchase_request_line to_update
+        SET qty_in_progress = 0.0
+        WHERE qty_in_progress is null
```

Comment:
change this query by:

`ALTER TABLE purchase_request_line ALTER COLUMN qty_in_progress DROP DEFAULT`